src/labels.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  fetchMergedWithReviewedWaitMerge,
+  removeReviewedWaitMergeLabel,
+} from "./github.ts";
+
+export const run = async () => {
+  const prsThatAreMergedAndHaveTheReviewedWaitMergeLabel =
+    await fetchMergedWithReviewedWaitMerge();
+  await Promise.all(
+    prsThatAreMergedAndHaveTheReviewedWaitMergeLabel.items.map(remove),
+  );
+};
+
+const remove = async (pr: { title: string; number: number }) => {
+  const response = await removeReviewedWaitMergeLabel(pr.number);
+  if (response.ok) {
+    console.log(`Removed reviewed/wait-merge from ${pr.title} (#${pr.number})`);
```

Comment:
```suggestion
    console.info(`Removed reviewed/wait-merge from ${pr.title} (#${pr.number})`);
```

I have the habit that I use `console.log` only for debug and `console.info` for non-debug. if you set up eslint like in the gitea repo, it'll warn for the former, but not the latter.